rules/windows/initial_access_suspicious_ms_exchange_process.toml

```
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/03/04"
+maturity = "production"
+updated_date = "2021/03/04"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies suspicious processes being spawned by the Microsoft Exchange Server Unified Messaging (UM) service. This
+activity has been observed exploiting CVE-2021-26857.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Microsoft Exchange Server UM Spawning Suspicious Processes"
+references = [
+    "https://www.microsoft.com/security/blog/2021/03/02/hafnium-targeting-exchange-servers",
+    "https://www.volexity.com/blog/2021/03/02/active-exploitation-of-microsoft-exchange-zero-day-vulnerabilities",
+]
+risk_score = 47
+rule_id = "483c4daf-b0c6-49e0-adf3-0bfa93231d6b"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+  process.parent.name : ("UMService.exe", "UMWorkerProcess.exe") and
+    not process.parent.name : ("werfault.exe", "wermgr.exe")
```

is this supposed to be `process.name`?
```suggestion
    not process.name : ("werfault.exe", "wermgr.exe")
```

---

This will trigger on any legitimate spawned processes as well without tighter logic or exlcusions/inclusions. Can we at least add some FP verbiage and precautions around that?

---

Yeh should be process.name..

---

should be good to go now

---

lol, no more hot tubs for murph